fire/cli/info.py

```diff
@@ -474,13 +499,22 @@ def infotype(infotype: str, **kwargs):
         fire.cli.print(f"  Name        :  {pit.name}")
         fire.cli.print(f"  Description :  {pit.beskrivelse}")
         fire.cli.print(f"  Type        :  {pit.anvendelse}")
+        return
+
+    if infotype == "":
+        fire.cli.print("Følgende punktinfotyper er tilgængelige:\n")
     else:
-        if infotype == "":
-            fire.cli.print("Følgende punktinfotyper er tilgængelige:\n")
-        else:
-            fire.cli.print(f'"{infotype}" matcher følgende punktinfotyper:\n')
+        fire.cli.print(f'"{infotype}" matcher følgende punktinfotyper:\n')
+
+    if not søg:
```

Comment:
vil det her betyde at når du giver en infotype (og ikke bruger søg funktionen) så vil der blive vist en liste med tilgængelige infotyper?

Reply:
Ja - det var den eksisterende søgefunktionalitet, som @kbevers introducerede for noget tid siden:

```sh
$ fire info infotype IDENT   
"IDENT" matcher følgende punktinfotyper:
                                        
IDENT:GI                                
IDENT:GNSS                              
IDENT:diverse                           
IDENT:ekstern                           
IDENT:jessen                            
IDENT:landsnr                           
IDENT:refgeo_id                         
IDENT:station                           
```

og 

```sh
$ fire info infotype         
Følgende punktinfotyper er tilgængelige:

AFM:1000
AFM:1001
AFM:1002
... (etc ad nauseam)
```

men den matcher kun på (starten af) infotypenavnet (derfor matcher en tom infotype alt), ikke på beskrivelsen - og det er lidt svært at huske hvilken afmærkningstype, der svarer til hvilken `AFM:nnnn`. Derfor denne udvidelse.

Reply:
ok, så forstår jeg. Så ser det hele ud til at give mening